AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift

```diff
@@ -36,50 +36,66 @@
     /// Handles the 3D Secure 2 fingerprint and challenge actions separately + Delegated Authentication.
     @available(iOS 14.0, *)
     internal class ThreeDS2PlusDACoreActionHandler: ThreeDS2CoreActionHandler {
-        
+    
         internal var delegatedAuthenticationState: DelegatedAuthenticationState = .init()
         
         internal struct DelegatedAuthenticationState {
             internal var isDeviceRegistrationFlow: Bool = false
         }
-        
+    
         private let delegatedAuthenticationService: AuthenticationServiceProtocol
-
+        private let deviceSupportCheckerService: AdyenAuthentication.DeviceSupportCheckerProtocol
+        private let presenter: ThreeDS2PlusDAScreenPresenterProtocol
+        
         /// Initializes the 3D Secure 2 action handler.
         ///
         /// - Parameter context: The context object for this component.
         /// - Parameter appearanceConfiguration: The appearance configuration.
+        /// - Parameter style: The delegate authentication component style.
         /// - Parameter delegatedAuthenticationConfiguration: The delegated authentication configuration.
+        /// - Parameter presentationDelegate: The presentation delegate
         internal convenience init(
             context: AdyenContext,
             appearanceConfiguration: ADYAppearanceConfiguration,
-            delegatedAuthenticationConfiguration: ThreeDS2Component.Configuration.DelegatedAuthentication
+            delegatedAuthenticationConfiguration: ThreeDS2Component.Configuration.DelegatedAuthentication,
+            presentationDelegate: PresentationDelegate?
         ) {
```

Comment:
There is no `style` param on `init`, but there is a meta-doc.

Reply:
Nice catch.  thanks. ✅